src/material-experimental/mdc-tabs/tab-group.ts

```diff
@@ -62,7 +62,10 @@ export class MatTabGroup extends _MatTabGroupBase {
   /** Whether the ink bar should fit its width to the size of the tab label content. */
   @Input()
   get fitInkBarToContent(): boolean { return this._fitInkBarToContent; }
-  set fitInkBarToContent(v: boolean) { this._fitInkBarToContent = coerceBooleanProperty(v); }
+  set fitInkBarToContent(v: boolean) {
+    this._fitInkBarToContent = coerceBooleanProperty(v);
+    this._changeDetectorRef.markForCheck();
```

Comment:
We decided in a past team meeting to explicitly *not* add `markForCheck` to MDC setters

Reply:
Talked with Igor, his opinion is that this is correct to call `markForCheck` here, especially since the client would not know that it needs to

Reply:
When we talked about this in the team meeting we talked about two scenarios:

&nbsp; A. Input set through template.
&nbsp; A. Input set programmatically.

To cover (A), we should not need `markForCheck`, but for handling (B) we need to add `markForCheck`. Is the intention here to handle the second scenario here?

If we do want to handle (B), then I think it doesn't make sense to only do that for boolean inputs. It would be inconsistent with inputs not doing coercion. The decision (unless I'm wrong) was based on the idea that we do not want to introduce such an inconsistent public API _again_ (where we handle _A_ but only partially _B_).

Reply:
In this case, this is addressing an issue for (B).

It does seem to me that we should be doing `markForCheck` for any `Input()` that may need it (regardless if it's Boolean).

Reply:
I added an item for the next meeting agenda to (re)discuss it

Reply:
There is a desire to discuss `OnPush` and programmatic changes to `Input()`s between parent/child components in a meeting with the framework team.